### ISSUE TITLE
Licensing of Project Carrot based code

### ISSUE THREAD

**Issue by soulweaver91** [deathkiller/jazz2#1]
Hi!

It's good to see more JJ2 projects crop up, and it seems you have had time to put together an even more thorough version than I had so far. That's great! That said, I'd hate to be a stickler, but I think I have to point out that you might inadvertedly not be following PC's licensing properly. The MIT license states:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

There are several files in this project that are directly ported from PC, just rewritten in C#; I believe these all should be noted with the MIT license including the copyright string as the logic behind the code is still a "substantial portion" of PC. However, the only file I can find at least the latter is in the about window. I believe that in order to be not be in breach of the license, you should add that license comment on each directly derived file.

TL;DR: It's not like there will be any legal consequences, since that'd just be bonkers, but I do think the acknowledge of my efforts behind your project is tucked away a bit too far aside considering the amount of code you borrowed :p

**Comment by deathkiller** [comment]
Hi,

I can't add license comment on each directly derived file, because I don't remember which files were directly derived, all of them had to be rewritten... That's why there is notice in About menu. Additionally, many of them were written from scratch, but can look similar to yours.

I can include your LICENSE file with statement that some of the files were derived from your work, if the mention in About menu is not enough, but I'm afraid I can't do much more...
